dotnet/src/SemanticKernel.Skills/Skills.Memory.VectorDB/Qdrant/DotNet/Internal/Diagnostics/ConsoleLogger.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I think this should be deleted and updated to use the logger of the Kernel.

Reply:
Take in an ILogger (and likely a user would then supply the logger that is used by the kernel).